### PR TITLE
chore(storage): Upgrading ruby to ruby 3

### PR DESCRIPTION
**Pull request by shubhangi-google**
upgrading ruby version dependency for Google cloud storage
sending this as a minor version release

### CODE REVIEW

**Review by bajajneha27 — APPROVED** [review]
LGTM from my end.

File: google-cloud-storage/lib/google/cloud/storage/version.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
 module Google
   module Cloud
     module Storage
-      VERSION = "1.52.0".freeze
```
Comment (dazuma) [inline google-cloud-storage/lib/google/cloud/storage/version.rb:19]: Please do not update the version manually. This is read and updated during the release process, and modifying it manually will confuse the releaser.

File: google-cloud-storage/README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Google::Apis.logger = my_logger
 
 ## Supported Ruby Versions
 
-This library is supported on Ruby 2.5+.
+This library is supported on Ruby 3.0.0+.
 
 Google provides official support for Ruby versions that are actively supported
 by Ruby Core—that is, Ruby versions that are either in normal maintenance or in
```
Comment (dazuma) [inline google-cloud-storage/README.md:66]: You'll need to update the following line as well, as it references Ruby 2.5.

**Review by dazuma — CHANGES_REQUESTED** [review]

**Review by bajajneha27 — APPROVED** [review]

**Comment by bajajneha27** [comment]
@dazuma Shivangi has made the changes requested. Can you please take a look?

**Review by dazuma — APPROVED** [review]
Looks good, thanks!

**Pull request merged**